src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1519,4 +1519,16 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="UseStringContainsCharOverloadWithSingleCharactersDescription" xml:space="preserve">
+    <value>String.Contains(String) internally uses an obsolete inner loop when searching for a single character. String.Contains(char) will internally use String.IndexOf(char).</value>
+  </data>
+  <data name="UseStringContainsCharOverloadWithSingleCharactersMessage" xml:space="preserve">
+    <value>Consider using String.Contains(char) instead of String.Contains(String) when searching for a single character</value>
```

Comment:
```suggestion
    <value>Consider using 'string.Contains(char)' instead of 'string.Contains(string)' when searching for a single character</value>
```